src/mimeparser.rs

```diff
@@ -1029,6 +1046,12 @@ impl MimeMessage {
                 })
                 .ok();
             return;
+        } else if filename == "status-update.json" {
```

Comment:
Is there a specification for what you think this status update json will be? I can imagine a world where it won't be a json file

Reply:
> Is there a specification for what you think this status update json will be?

no. currently, it is just an array of payloads the app has given to `sendUpdate()`, sth. as `[{payload: foo}, {payload: bar}]`. i think, this "wire" thingie is nothing the apps really have to worry about, and, in, fact, we could have chosen any other format here (json was just handy as this is the same format js will receive at the end :)